src/main/java/me/moodcat/core/UnauthorizedException.java

```diff
@@ -0,0 +1,13 @@
+package me.moodcat.core;
+
+/**
+ * Thrown when the user is not logged in.
+ */
+public class UnauthorizedException extends Exception {
```

Comment:
Thre already is a NotAuthorisedException in the dependencies which is used for authorization. Please conform to that exception.